forge-gui/res/cardsfolder/upcoming/unlicensed_hearse.txt

```diff
@@ -0,0 +1,14 @@
+Name:Unlicensed Hearse
+ManaCost:2
+Types:Artifact Vehicle
+PT:*/*
+A:AB$ ChangeZone | Cost$ T | TargetMin$ 0 | TargetMax$ 2 | TargetsFromSingleZone$ True | Origin$ Graveyard | Destination$ Exile | SubAbility$ DBStore | TgtPrompt$ Select up to two target cards in a graveyard | ValidTgts$ Card | RememberChanged$ True | SpellDescription$ Exile up to two target cards from a single graveyard.
+SVar:DBStore:DB$ StoreSVar | SVar$ ExiledAmount | Type$ CountSVar | Expression$ ExiledAmount/Plus.X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+S:Mode$ Continuous | Affected$ Card.Self | SetPower$ ExiledAmount | SetToughness$ ExiledAmount | Description$ CARDNAME's power and toughness are each equal to the number of cards exiled with it.
```

Comment:
"exiled with" means cards in exile itself exiled it
that also means that if the exiled card leaves exile, it isn't exiled with anymore

(that later needs to be connected to linked ability anyway)

Reply:
It will be great to streamline all these ExiledWithSource+Remembered cards